tests/Resources/ti.ui.textfield.test.js

```diff
@@ -398,6 +398,32 @@
 			should(textFieldA.hasText()).be.true();
 			should(textFieldB.hasText()).be.true();
 		});
+
+		it('#setSelection', function (finish) {
+			this.timeout(5000);
+			const textField = Ti.UI.createTextField({
+				value: 'Lorem ipsum dolor sit amet.',
+				width: Ti.UI.SIZE
+			});
+			win = Ti.UI.createWindow({
+				backgroundColor: '#ddd'
+			});
+			win.add(textField);
+			win.addEventListener('postlayout', function listener () {
+				win.removeEventListener('postlayout', listener);
+				textField.setSelection(0, 5);
+				setTimeout(function (e) {
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.textfield.test.js;
  Line: 415;
-->

- :warning: tests/Resources/ti.ui.textfield.test.js line 415 – 'e' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)